Gruntfile.js

```diff
@@ -83,7 +83,10 @@ module.exports = function (grunt) {
 
     propertiesToJSON: {
       main: {
-        src: ['resources/nls/*.properties'],
+        src: [
```

Comment:
In the case of the sign-in-widget, we can actually get rid of most of the i18n logic (differs from the plugin). Instead of using the .properties files, we should use the exported requireJs folder.

When we switch to using that, we can get rid of these targets in the Gruntfile:
- propertiesToJson
- JSONToJs

As well as the corresponding dependency (grunt-properties-to-json) and buildtools/JSONToJs